app/assets/javascripts/image_question_drawing_tool.js.coffee

```diff
@@ -74,6 +73,10 @@ class ImageQuestionDrawingTool
     @$annotation_field          = $("#{@form_sel} [name=\"#{@form_prefix}[annotation]\"]")
     @$answer_text_field         = $("#{@form_sel} [name=\"#{@form_prefix}[answer_text]\"]")
 
+    # Find the first interactive which is on the same page (aka content-module, div.content-mod) as the question.
+    # It's necessary when we are in single page rendering mode and there are multiple interactives rendered.
+    @interactive_element = @$main_form.closest('.content-mod').find('.interactive-mod > *:first-child')[0]
```

Comment:
Makes sense to me, updated: https://github.com/concord-consortium/lara/commit/9f22885ae19fa124f9e50e0c79f4ba2df80a7709
I also added very simple handling of one edge case (alert when interactive is not defined).